collect_app/src/main/java/org/odk/collect/android/utilities/StringUtils.java

```diff
@@ -190,5 +192,13 @@ public static String removeEnd(String str, String remove) {
 
         return str;
     }
+
+    public static String getFilenameError(String filename) {
+        String possiblyRestricted = "?:\"*|/\\<>\u0000";
```

Comment:
You don't wan't to log the whole name for privacy reasons or what?

Reply:
Yeah, @lognaturel and I chatted last night, and we wanted to avoid that.